README.md

```diff
@@ -20,10 +20,10 @@ For support requests, please open up a [bug issue](https://github.com/gnosis/saf
 
 ## Deployed environments
 
-* Production: https://gnosis-safe.io/app/
-* Staging: https://safe-team.staging.gnosisdev.com/app/
-* Dev: https://safe-team.dev.gnosisdev.com/app/
-* PRs: https://pr<PR_NUMBER>--safereact.review-safe.gnosisdev.com/app/
+- Production: https://gnosis-safe.io/app/
```

Comment:
Prettier automatically altered this. Please let me know if it should be reverted.

Reply:
It's fine. Since you're committing this, could you also wrap the last link (PRs) into a code block? Otherwise it looks weird on GH.